src/index.html

```diff
@@ -7,8 +7,92 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;500;700&family=Roboto:wght@500&family=Splash&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="top">
+    <header class="header">
+      <a
+        class="image__logo"
+        href="#top"
+      >
+        <img
+          src="images/logo.png"
+          alt="img__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              id="first_link"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#mobiles"
+            >
```

Comment:
```suggestion
            <a class="nav__link" href="#mobiles">
```
DOn't need to start each attribute from the new line if tag has only two attributes.

Reply:
it's much more comfortable for me to read code in this style. Plus, if we add new attributes it will be unnecessary to move the code on a large number of lines. I think so